### PR TITLE
Removing Cyclic Redundancy

### PR DESCRIPTION
**Pull request by ShivamNegi**
Taking inputs from #825 . This commit works on removing cyclic redundancies. @henrykironde please review.

### CODE REVIEW

File: retriever/engines/msaccess.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-from __future__ import print_function
```
Comment (henrykironde) [inline retriever/engines/msaccess.py]: from __future__ import print_function
This is needed for p3

Comment (ShivamNegi) [inline retriever/engines/msaccess.py]: what is p3?

Comment (ethanwhite) [inline retriever/engines/msaccess.py]: Python 3, thought I think there's a typo and @henrykironde meant Python 2 in this case. Without the import lines that use print like a function `print(...)` will error in Python 2.

Comment (ShivamNegi) [inline retriever/engines/msaccess.py]: updated.

**Pull request merged**